scripts/observability/manage-local-openobserve.sh

```
@@ -10,10 +10,36 @@ DOCTOR_SCRIPT="${SCRIPT_DIR}/check-proxy-telemetry.mjs"
+      key="${key#"${key%%[![:space:]]*}"}"
+      key="${key%"${key##*[![:space:]]}"}"
+      value="${value#"${value%%[![:space:]]*}"}"
+      value="${value%"${value##*[![:space:]]}"}"
+
+      if [[ "${value}" =~ ^\".*\"$ || "${value}" =~ ^\'.*\'$ ]]; then
+        value="${value:1:${#value}-2}"
+      fi
+
+      if [[ -n "${key}" ]]; then
+        printf -v "${key}" '%s' "${value}"
+        export "${key}"
```

Shellcheck (0.11.0)

[warning] This does not export 'key'. Remove $/${} for that, or use ${var?} to quiet.

(SC2163)